src/Raven.Server/Documents/ETL/Providers/Queue/Handlers/QueueEtlConnectionHandler.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Confluent.Kafka;
+using Newtonsoft.Json;
+using RabbitMQ.Client;
+using Raven.Client.Documents.Operations.ETL.Queue;
+using Raven.Server.Routing;
+using Raven.Server.Web.System;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Server.Documents.ETL.Providers.Queue.Handlers
+{
+    public class QueueEtlConnectionHandler : DatabaseRequestHandler
+    {
+        [RavenAction("/databases/*/admin/etl/queue/test-connection/kafka", "POST", AuthorizationStatus.DatabaseAdmin)]
```

Comment:
should't this be the other way around?
/databases/*/admin/etl/queue/test-connection/kafka => /databases/*/admin/etl/queue/kafka/test-connection

Reply:
done